metadata.json

```diff
@@ -1,7 +1,8 @@
 {
     "shell-version": [
         "40",
```

Comment:
Did you ensure everything works with 40/41 as well?

Reply:
This is a good point. I will remove these, just in case. Changes like these are often not backward compatible with Gnome.